.github/workflows/create_release.yml

```diff
@@ -45,6 +50,11 @@ env:
         required: false
         default: false
         type: boolean
+      only-docker:
```

Comment:
How terrible that we need so many duplicates here and there https://github.com/actions/runner/issues/1182#issue-935195181